src/style.css

```
@@ -1 +1,8 @@
-/* styles go here */
+.form-field {
+  display: block;
+  margin: 10px 0;
```

```suggestion
  margin-bottom: 10px;
```
You could use just `margin-bottom` instead of shorthand